ReplacingDocker/podman.sh

```diff
@@ -0,0 +1,293 @@
+#!/bin/sh
+
+# podmah.sh demo script.
+# This script will demonstrate a lot of the features of podman, concentrating
+# on the security features.
+
+buildah_image() {
+    sudo podman images  | grep -q -w buildah-ctr
+    if [[ $? != 0 ]]; then
+	sudo podman build -t buildah-ctr -f Dockerfile.buildah .
+    fi
+}
+
+setup() {
+    sudo cp /usr/share/doc/audit/rules/10-base-config.rules /etc/audit/rules.d/audit.rules
+    sudo augenrules --load > /dev/null
+    sudo systemctl restart auditd 2> /dev/null
+    sudo systemctl restart docker
+    clear
+}
+version() {
+    # Podman inside a container
+    read -p "sudo podman version"
+    echo ""
+    sudo podman version
+    echo ""
+
+    # Podman inside a container
+    read -p "sudo podman info"
+    echo ""
+    sudo podman info
+}
+
+rootless() {
+    # Rootless podman
+    read -p "Podman as rootless"
+    echo ""
+
+    read -p "--> podman pull alpine"
+    podman pull alpine
+    echo ""
+
+    read -p "--> podman run --net=host --rm alpine ls"
+    podman run --net=host alpine ls
+    echo ""
+
+    echo "Show Non Privileged containers"
+    read -p "--> podman ps -a"
+    podman ps -a
+    echo ""
+
+    echo "Show Non Privileged images"
+    read -p "--> podman images"
+    podman images
+    echo ""
+
+    echo "Show Privileged images"
+    read -p "--> sudo podman images"
+    sudo podman images
+    echo ""
+
+    read -p "--> clear"
+    clear
+}
+
+userns() {
+    read -p "--> buildah unshare"
+    buildah unshare
+    echo ""
+
+    read -p "--> clear"
```

Comment:
I think it's expected, but I need to do `exit` here, might want to display that in the read for the unshare.